### PR TITLE
Opam publish bitlib.0.1.1

### PR DESCRIPTION
**Pull request by chrisnevers**

### CODE REVIEW

File: packages/bitlib/bitlib.0.1.1/opam [hunk 1]
```diff
@@ -0,0 +1,16 @@
+opam-version: "2.0"
+synopsis: "A library for writing binary files"
+maintainer: "Chris Nevers <christophernevers96@gmail.com>"
+authors: "Chris Nevers <christophernevers96@gmail.com>"
+homepage: "https://github.com/chrisnevers/bitlib"
+bug-reports: "https://github.com/chrisnevers/bitlib/issues"
+depends: ["ocaml" "ocamlbuild"]
+build: make
```
Comment (mseri) [inline packages/bitlib/bitlib.0.1.1/opam]: Looking at the code, it seems to me that you are only building the library and don’t install anyhing

Comment (chrisnevers) [reply packages/bitlib/bitlib.0.1.1/opam]: Thanks for spotting that, I am a complete beginner and would appreciate any help in getting this working/prepared. What do I have to do?

Comment (mseri) [reply packages/bitlib/bitlib.0.1.1/opam]: I think the easiest way to make sure that you buil and install the library is to use dune as build system

**Comment by kit-ty-kate** [comment]
I'm guessing this PR is superseded by https://github.com/ocaml/opam-repository/pull/14291. Closing then

**Pull request closed without merging**